package.json

```diff
@@ -8,6 +8,7 @@
     "lint": "./scripts/lint-new-work.sh",
     "lint-all": "eslint client/\\**/*.ts server/\\**/*.ts",
     "listen": "RUN_AS_LISTENER=true ts-node --project tsconfig.node.json server.ts",
+    "load-tokens": "SHOULD_LOAD_DATABASE_FROM_TOKEN_LISTS=true nodemon --delay 0.25 --watch 'server/**/*.ts' --ignore 'server/**/*.spec.ts' --exec 'ts-node --project tsconfig.node.json' server.ts",
```

Comment:
This shouldn't use nodemon because it's a one-off script and not a daemon like the main server process.